pkg/kv/kvserver/replica_init.go

```diff
@@ -124,7 +124,15 @@ func newUnloadedReplica(
 	// Add replica pointer value. NB: this was historically useful for debugging
 	// replica GC issues, but is a distraction at the moment.
 	// r.AmbientContext.AddLogTag("@", fmt.Sprintf("%x", unsafe.Pointer(r)))
+
 	r.raftMu.stateLoader = stateloader.Make(desc.RangeID)
+	r.raftMu.sideloaded = logstore.NewDiskSideloadStorage(
```

Comment:
No need to change anything, just wanted to comment on this bit from the commit msg:

> Sideload storage is necessary both for initialized and uninitialized replicas,

It's technically not necessary for uninited replicas. Sideloading only matters when you have a nontrivial log, and uninited by definition don't. But practically it's difficult to structure the code such that we don't have awkward exceptions for uninited replicas since they share all code paths with inited onces. So this is fine.

Reply:
Yep, I agree. And caught a few of exceptions/panics in uninited replicas.